pkg/config/config.go

```diff
@@ -155,6 +155,8 @@ type Config struct {
 	SquashFsCompressionConfig []string              `yaml:"squash-compression,omitempty" mapstructure:"squash-compression"`
 	SquashFsNoCompression     bool                  `yaml:"squash-no-compression,omitempty" mapstructure:"squash-no-compression"`
 	UkiMaxEntries             int                   `yaml:"uki-max-entries,omitempty" mapstructure:"uki-max-entries"`
+	BindPCR                   []string              `yaml:"bind-pcr,omitempty" mapstructure:"bind-pcr"`
```

Comment:
use plural since it's a slice? `BindPCRs` and `BindPublicPCRs` below?

Reply:
ah good point